frontend/src/tests/offer-tests.js

```diff
@@ -74,9 +74,30 @@ export function offersTests(api) {
         expect(newOffer.emailed_date).not.toBeNull();
     });
 
-    it.todo(
-        "when an offer is emailed a second time, the emailed date is updated to the most recent emailed date"
-    );
+    it("when an offer is emailed a second time, the emailed date is updated to the most recent emailed date", async () => {
+        // record previous emailed_date
+        const resp = await apiGET(
+            `/admin/assignments/${assignment.id}/active_offer`
+        );
+        expect(resp).toHaveStatus("success");
+        let previousDate = resp.payload.emailed_date;
+
+        // send the email again
+        const resp1 = await apiPOST(
+            `/admin/assignments/${assignment.id}/active_offer/email`
+        );
+        expect(resp1).toHaveStatus("success");
+        // check if date is updated
+        newOffer = resp1.payload;
+        expect(newOffer.emailed_date).not.toEqual(previousDate);
+
```

Comment:
Preferably to make things more consistent by 
1) saving 'resp.payload' into a variable named 'previousOffer'
or 
2) saving 'resp1.payload.emailed_date' as a new variable named 'newDate'

Reply:
thank you! will fix this! :)